### DISCUSSION TITLE
How do I modify php dap config without getting two php dap configs?

### DISCUSSION THREAD

**Discussion by gmnz** [LazyVim/LazyVim#4528] [Q&A]
Hi,

So I guess the question is pretty clear from the title, but just to be sure, here are some screenshots of what I mean:

Result of my change:

![image](https://github.com/user-attachments/assets/98ad03ae-090e-463b-beaa-489c2097f092)

My config change:

![image](https://github.com/user-attachments/assets/1b7359ff-7f02-4b44-9b86-0ce8bc1f3164)

I actually just wanted to change the port, not to create another php dap config. It's annoying that it's asking me which I want to use every time I need to debug. Any ideas? I have php config from LazyExtras on.

**Comment by dpetka2001** [comment]
Try adding 
```lua
  {
    "jay-babu/mason-nvim-dap.nvim",
    opts = {
      handlers = {
        php = function() end,
      },
    },
  },
```
Not sure if that does what you want or not. I don't actually use `nvim-dap`.

**Reply by gmnz** [reply]
Cool, it works. Thanks a lot!
